### DISCUSSION TITLE
先前训练的模型可以删除吗

### DISCUSSION THREAD

**Discussion by bilbillm** [svc-develop-team/so-vits-svc#50] [Q&A] [answered]
现在训练到G9000，云盘空间不够了，可以把之前的D0 ~ D8000和G0 ~ G8000删除吗？
会不会对训练断点产生影响？

**Comment by HuanLinOTO** [comment, marked as answer]
会读取 step 数最大的模型，可以删，但是不建议，就怕过拟合了。

**Comment by Miuzarte** [comment]
当然可以，不过再往后的建议把模型放到本机来腾空间，免得过拟合没得救

**Comment by HuanLinOTO** [comment]
另外 你的问题比较适合去 discussions 问，issue 是负责提项目本身的问题的

**Comment by PriesiaMioShirakana** [comment]
建议至少留下两个最近的模型，因为最后一个模型如果在存储不足的情况下会保存不完全导致前功尽弃
